build-plugin/spring-boot-maven-plugin/src/intTest/java/org/springframework/boot/maven/AotTests.java

```diff
@@ -195,4 +195,39 @@ protected String buildLog(File project) {
 		return contentOf(new File(project, "target/build.log"));
 	}
 
+	@TestTemplate
+	void whenAotRunsWithDevtoolsInClasspathItIsExcluded(MavenBuild mavenBuild) {
+		mavenBuild.project("aot-exclude-devtools").goals("package").execute((project) -> {
+			// The test passes if the build completes successfully.
```

Comment:
Please remove those comments. We don't usually have verbose explanations like this.